packages/new-polymath-issuer/src/components/enhancers/__tests__/ParseCsv.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { render, cleanup, wait } from 'react-testing-library';
+import { ParseCsv, RenderProps } from '../ParseCsv';
+import { validators } from '@polymathnetwork/new-shared';
+
+afterEach(cleanup);
+
+describe('ParseCsv component', () => {
+  it('calls the render function', () => {
+    const renderMock = jest.fn(() => {
+      return <div>Hello</div>;
+    });
+    render(<ParseCsv parseErrorMessage="Parsing Error" render={renderMock} />);
+    expect(renderMock).toHaveBeenCalled();
+  });
+
+  it('sets the result when setFile is called', async () => {
+    let setFile: Function = () => null;
+    let clearFile: Function = () => null;
+    let result: Array<any> = [];
+    const renderFunction = (inputParams: RenderProps) => {
+      setFile = inputParams.setFile;
+      clearFile = inputParams.clearFile;
+      result = inputParams.data.result;
+      return <div>Hello</div>;
+    };
+    render(
+      <ParseCsv parseErrorMessage="Parsing Error" render={renderFunction} />
+    );
+
+    setFile({
```

Comment:
I think there was a confusion with how I suggested the column data was done. What I mean was that, instead of passing the configuration here, I suggested we did it in the component itself:

```ts
const fileConfig = {
  data: `column 1, column 2`
/* ... etc ... */
};

render (
  <ParseCsv config={fileConfig} {/* ... */} />
)
```

If we don't do this then our Presenter becomes aware of the file structure, which is something we shouldn't enforce